# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/*/straw/install-hotfix.patch
# Copyright (C) 1998 - 2004 ROCK Linux Project
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -urN straw.orig/Makefile straw/Makefile
--- straw.orig/Makefile	2003-11-25 11:22:22.000000000 +0100
+++ straw/Makefile	2003-11-27 07:11:26.000000000 +0100
@@ -1,4 +1,4 @@
-PYTHON ?= python2.2
+PYTHON ?= python2.3
 INSTALL ?= install
 RM ?= rm
 MSGFMT ?= msgfmt
@@ -11,14 +11,14 @@
 PREFIX ?= `pkg-config libgnome-2.0 --variable=prefix || echo /usr/local`
 
 # If you edited PYTHON above, edit this to the corresponding python lib dir
-LIBDIR = $(PREFIX)/lib/python2.2/site-packages/straw
+LIBDIR = $(PREFIX)/lib/python2.3/site-packages/straw
 BINDIR = $(PREFIX)/bin
 DATADIR = $(PREFIX)/share/straw
 APPLICATIONSDIR = $(PREFIX)/share/applications
 ICONDIR = $(PREFIX)/share/pixmaps
 LOCALEDIR = $(PREFIX)/share/locale
 # change this to just "/etc" if you're installing in /usr
-SYSCONFDIR = $(PREFIX)/etc
+SYSCONFDIR = /etc/$(PREFIX)
 
 POFILES := $(shell echo po/*.po)
 MOFILES := $(POFILES:.po=.mo)
@@ -27,8 +27,8 @@
 schema_DATA = data/straw.schemas 
 # modify this if you're running your own build environment (garnome/jhbuild)
 # change it to "xml://path/to/gnome_build_env/etc/gconf/gconf.xml.defaults"
-GCONF_SCHEMA_CONFIG_SOURCE = xml::/etc/gconf/gconf.xml.defaults 
-GCONF_SCHEMA_FILE_DIR = /etc/gconf/schemas/
+GCONF_SCHEMA_CONFIG_SOURCE = xml::$(SYSCONFDIR)/gconf/gconf.xml.defaults 
+GCONF_SCHEMA_FILE_DIR = $(SYSCONFDIR)/gconf/schemas/
 GCONFTOOL = gconftool-2
 
 SCHEMADIR = $(GCONF_SCHEMA_FILE_DIR)
diff -urN straw.orig/src/straw straw/src/straw
--- straw.orig/src/straw	2002-09-12 01:15:49.000000000 +0200
+++ straw/src/straw	2003-11-27 07:00:11.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2.2
+#!/usr/bin/env python
 #
 # Copyright (c) 2002 Juri Pakaste
 # You may use and distribute this software under the terms of the
